### ISSUE TITLE
[Bug] [a11y] Addons - Lag for a couple of seconds when interchanging buttons

### ISSUE THREAD

**Issue by Diana-Rus** [mozilla-mobile/fenix#8638]
Pre-requisites:
- Have TalkBack enabled.
- Have Ublock addon installed and enabled in Firefox Preview Beta.

## Steps to reproduce
1. Open Fenix and go to "Settings"/"Add-ons".
2. Tap on the u-Block Origin add-on.
3. Tap on the "Settings" option.
4. Choose option "Back up to file" from the second half of the screen.
5. Tap on the "Restore from file..." option.

### Expected behavior
The TalkBack is activated and describes the button label.

### Actual behavior
There is a lag until TalkBack is activated and describes the button label.

### Device information

* Android device: Google Pixel C (Android 8.0.0)
* Fenix version: Firefox Preview Beta 4.0.0-beta.1

**Comment by asadotzler** [comment]
This seems like a bug with the extension and not necessarily with Fenix. Am I misunderstanding the report?

**Comment by mcarare** [comment]
I cannot reproduce any significant a11y issue on this screen, and as in the comment above, I'm not sure I understand what the reported problem is. @Diana-Rus can you please, clarify? TY!

**Comment by softvision-miralobontiu** [comment]
I was not able to reproduce this issue on the 11/18 Nightly build, with Samsung Galaxy Tab S6 (Android 10).
I'll close this issue.
